components/AboutTitleBar.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+  <div class="AboutTitleBar">
+    <div class="Background-Wrapper">
+      <div class="header-container">
+        <h2>
+          <span class="headertitle">
+            About the Bay Area Pandemic Dashboard
+          </span>
+        </h2>
+        <p>
+          <span class="headersubtitle">
+            Local data and information associated with the coronavirus pandemic
+          </span>
```

Comment:
Similarly, paragraph tag can group the wordings without the need for a span, unless you need to specifically target a group of words.

Reply:
on latest commit